__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require("../app/app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const { topicData, userData, articleData, commentData } = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("GET from non-existent endpoint", () => {
+    test("404: sends an appropriate status and error message when given a non-existent endpoint", () => {
+      return request(app)
+        .get("/api")
+        .expect(404)
```

Comment:
no need to test - this is express's inbuild error.  I would only include a 404 test if you've used a custom 404 handler in the app.